### ISSUE TITLE
Membership stats graph is incorrect

### ISSUE THREAD

**Issue by NFarrington** [VATSIM-UK/core#349]
In gitlab by @A-Lawrence on Jan 11, 2016, 23:54

Labels "Members Division" as "Members Total" and vice versa.

**Comment by NFarrington** [comment]
In gitlab by @NFarrington on Jan 12, 2016, 17:37

@A-Lawrence addCurrentMembersStatistic() doesn't account for members with null created_at dates (of which there are many), hence why total members appears to be less than division members (which seem to be calculated from a different table, hence why the error isn't reproduced here)

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Feb 1, 2016, 21:24

I've manually fixed the null created_at timestamps.  I need to check if they start to occur again and then track down where it's happening (as it shouldn't be).

There are a _lot_ of past members that have a NULL joined_at date, because they've been suspended.  I don't think there's much we can do with that.

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Feb 1, 2016, 21:25

Query that was ran:

```mysql
UPDATE `mship_account`
SET `created_at` = `joined_at`
WHERE `created_at` IS NULL
AND `joined_at` <> '0000-00-00 00:00:00'
```

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Feb 1, 2016, 21:36

mentioned in commit 5abcb4d751ddbfff4f697a74d0c8f3aac32e3bb0

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Feb 1, 2016, 21:51

mentioned in commit 7a1190406e9bf396993d4ddbcd740daa6f2fb6d2

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Feb 1, 2016, 21:52

This should now be resolved in commit 5abcb4d7.  When the next version is deployed, all stats need to be recreated (readme updated: 7a119040)
